metadata-models/src/main/pegasus/com/linkedin/execution/ExecutionRequestSource.pdl

```diff
@@ -6,11 +6,6 @@ record ExecutionRequestSource {
   /**
   * The type of the execution request source, e.g. INGESTION_SOURCE
   */
-  @Searchable = {
-    "fieldName": "sourceType",
-    "fieldType": "KEYWORD",
-    "queryByDefault": false
-  }
```

Comment:
why remove this searchable here? i know you just added it in the previous PR so i'm going to assume it's good to remove